README.rst

```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
     python3 -m pymongoexplain <path/to/your/script.py> [PARAMS] [--optname OPTS]
 
 
+Limitations
+-----------
```

Comment:
Please leave a blank line after the title.

Reply:
Done.